src/sdk/namespace_cache.js

```diff
@@ -583,7 +583,16 @@ class NamespaceCache {
     }
 
     async delete_multiple_objects(params, object_sdk) {
-        const deleted_res = await this.namespace_hub.delete_multiple_objects(params, object_sdk);
+        const [hub_res, cache_res] = await Promise.allSettled([
+            this.namespace_hub.delete_multiple_objects(params, object_sdk),
```

Comment:
@paulxuzhang 
This breaks the trigger logic.
If you have a trigger it will not work properly.
Notice that later on, you run a query:
```obj_md = _.defaults({ key: obj.key }, await this.namespace_hub.read_object_md(request, object_sdk));```
There is no way that this query will find the object_md since you've already deleted it in ```Promise.allSettled```.
Unless I'm missing something.

Reply:
You are right. I will fix.

Reply:
@jeniawhite Fixed. Could you verify?

Reply:
@paulxuzhang 
Looks valid. 
Please run the code and make sure that it works, thank you.

Side question:
Is there a reason why you are reading the object_md from ```namespace_hub``` and not from ```namespace_nb``` or a merge between them?
Wouldn't a read from ```namespace_nb``` be quicker?

Reply:
Object gets cached only when it is read or uploaded through noobaa endpoint. Object can be uploaded or updated directly against hub without going through noobaa endpoint. So cache does not always have all the objects, hub is the ultimate truth.